.studio/automate-gateway

```diff
@@ -223,13 +223,16 @@ function send_chef_run_failure_example() {
   install_if_missing core/curl curl
   install_if_missing core/jq-static jq
 
-  uuid=$(uuidgen)
+  local run_uuid=$(uuidgen)
+  local node_uuid=$(uuidgen)
+  local node_name="node_failed_$(($RANDOM % 100))"
+  local rfc_time=$(date --rfc-3339=seconds  -d "$(($RANDOM % 144)) hour ago" | sed 's/ /T/' | sed 's/\+.*/Z/')
```

Comment:
Some of the default formatting probably depends on the ENV, it'd probably be less error prone to use `date`'s formatting options to get what you want directly.